lesson3/ToDoList2/app/src/main/java/erica/todolist2/MainActivity.java

```diff
@@ -16,20 +16,18 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);                                                     //gets the layout for the view from activity main
-        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);                                     //defines the toolbar
-        setSupportActionBar(toolbar);                                                               //makes the toolbar
+        setContentView(R.layout.activity_main);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
 
         final Fragment listFrag = new ListFragment();
-        replaceFragment(listFrag);                                                                  //calls the replaceFragment method defined below for the current view to be the list fragment
+        replaceFragment(listFrag); // default view is the list fragment
 
-        Button button = (Button) findViewById(R.id.button);                                         //makes the button
-        button.setOnClickListener(new View.OnClickListener() {
+        Button listFragmentButton = (Button) findViewById(R.id.button);
+        listFragmentButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                replaceFragment(listFrag);                                                          //when the button is clicked view goes back to list fragment
-
-
+                replaceFragment(listFrag); // Go back to the list fragment
```

Comment:
Instead of saying all that, assume that the reader knows we're in the `onClick` function, so just explain that functionality here: go back to the list fragment